dynamiq/nodes/tools/function_tool.py

```diff
@@ -17,15 +17,15 @@ class FunctionTool(Node, Generic[T]):
     """
 
     group: Literal[NodeGroup.TOOLS] = NodeGroup.TOOLS
-    name: str = Field(default="Function Tool")
+    name: str = Field(default="function-tool")
```

Comment:
Field is obsolete here:
```suggestion
    name: str ="function-tool"
```